### DISCUSSION TITLE
Is it possible to generate data for multiple states at once?

### DISCUSSION THREAD

**Discussion by fabkury** [synthetichealth/synthea#1189] [Q&A] [answered]
I wanted to make a nationwide (USA) dataset. I know I could run Synthea once per state, but the IDs (e.g. patient ID) would collide (coincide).

Is there a way to obtain a multi-state dataset, without having to address ID collisions post-hoc?

**Comment by jawalonoski** [comment, marked as answer]
Use a different `seed` and `clinician seed` (using `-s` and `-cs` in the command line parameters) when you run each state.

Given that each ID is a UUID, they are extremely unlikely to conflict. See https://en.wikipedia.org/wiki/Universally_unique_identifier

**Reply by jawalonoski** [reply]
In the cases where they do conflict (for example, `Organizations`) it is because they are designed to conflict. In other words, if two Organizations have the same ID, they are the same organization.

**Comment by fabkury** [comment]
Thanks so much for the quick response and the explanations. I confess I didn't know they were UUIDs. I had imagined autoincrement IDs, because I am looking at the data through the result of https://github.com/OHDSI/ETL-Synthea, not the CSVs produced by Synthea itself. I apologize for the confusion.
